SimpleJavaProject/src/main/java/arina_tolchenitsyna/second_java_task/numbers_main/NumbersMain.java

```diff
@@ -0,0 +1,42 @@
+package arina_tolchenitsyna.second_java_task.numbers_main;
+
+import java.util.Random;
+
+public class NumbersMain {
+    public static void main(String[] args) {
+        int number = 9;
+        int squareRoot = (int) Math.sqrt(number);
+        System.out.println("Number is: " + number + " The root of this number is: " + squareRoot);
+
+        Random random = new Random();
+        int newRandom = random.nextInt(100);
```

Comment:
nextInt returns  value between 0 (inclusive) and the specified value (exclusive) 
so the range will be 0 - 99